apps/admin-gui/src/app/shared/components/dialogs/edit-email-footer-dialog/edit-email-footer-dialog.component.html

```diff
@@ -15,12 +15,21 @@ <h1 mat-dialog-title>{{'DIALOGS.NOTIFICATIONS_EDIT_FOOTER.TITLE' | translate}}</
               {{'DIALOGS.NOTIFICATIONS_EDIT_FOOTER.FORMAT_' + format | uppercase | translate}}
             </span>
           </ng-template>
-          <mat-form-field class="w-100">
+          <mat-form-field class="w-100" subscriptSizing="dynamic">
             <textarea *ngIf="format === 'plain_text'" [(ngModel)]="mailFooter" matInput rows="5">
             </textarea>
-            <textarea *ngIf="format === 'html'" [(ngModel)]="htmlMailFooter" matInput rows="5">
+            <textarea *ngIf="format === 'html'" [formControl]="htmlInput" matInput rows="5">
             </textarea>
+            <mat-error *ngIf="htmlInput.invalid && format==='html'">
+              {{htmlInput.errors.invalidHtmlContent}}
+            </mat-error>
           </mat-form-field>
+          <div class="mt-2 font-italic text-muted" *ngIf="format === 'plain_text'">
+            {{'DIALOGS.NOTIFICATIONS_EDIT_FOOTER.DESCRIPTION' | translate}}
+          </div>
+          <div class="mt-2 font-italic text-muted" *ngIf="format === 'html'">
+            {{'DIALOGS.NOTIFICATIONS_EDIT_FOOTER.DESCRIPTION_HTML' | translate}}
+          </div>
```

Comment:
As you added these lines with dynamic text, you should also delete an old div with static text:
```
<div class="mt-2 font-italic text-muted">
    {{'DIALOGS.NOTIFICATIONS_EDIT_FOOTER.DESCRIPTION' | translate}}
</div>
```
![image](https://user-images.githubusercontent.com/70371638/224995616-bf6f6103-eeac-4e58-af9c-b24610183ce4.png)